scripts/examples/01-Camera/03-Event-Cameras/02-Genx320/genx320_histogram_mode_grayscale_led_tracking.py

```diff
@@ -9,24 +9,25 @@
 # Prophesee active marker board with LEDs and the corresponding event-based data are shown in
 # https://youtu.be/j-LpkDpCxUU?si=jA3B4xZg9RHlyoW3.
 
-import sensor
+import csi
 import time
 
-sensor.reset()
-sensor.set_pixformat(sensor.GRAYSCALE)  # Must always be grayscale.
-sensor.set_framesize(sensor.B320X320)  # Must always be 320x320.
-sensor.set_brightness(128)  # Leave at 128 generally (this is the default).
-sensor.set_contrast(16)  # Increase to make the image pop.
-sensor.set_framerate(200)
+csi0 = csi.CSI(cid=csi.GENX320)
+csi0.reset()
+csi0.pixformat(csi.GRAYSCALE)  # Must always be grayscale.
+csi0.framesize(csi.B320X320)  # Must always be 320x320.
```

Comment:
I removed all these custom resolution constants in https://github.com/openmv/openmv/pull/2804 let's merge that first and then just use the w/h directly.

```suggestion
csi0.framesize((320, 320))
```

Reply:
Done, testing everything now.